presto-product-tests/conf/presto/etc/jvm.config

```diff
@@ -10,8 +10,6 @@
 -XX:+UseG1GC
 -XX:+UseGCOverheadLimit
 -XX:+ExplicitGCInvokesConcurrent
--XX:+CMSClassUnloadingEnabled
```

Comment:
Commit msg doesn't conform (sentence case). Btw, what about "Do not use unrecommended JVM opts in product tests" ?

Reply:
Does this option do anything with G1? Assuming it does not, then removing `AggressiveOpts` is the only real change here, so we should do that separately with a commit message to reflect it. Something like 
*"Remove AggressiveOpts JVM flag for product tests"*

Reply:
I just checked and we don't use `AggressiveOpts` on our production clusters, either

Reply:
It was the agent that caused failure.

Reply:
done